Analysis/Tasks/PWGLF/cascadebuilder.cxx

```diff
@@ -91,59 +91,60 @@ struct cascadeprefilterpairs {
   Configurable<double> v0radius{"v0radius", 0.9, "v0radius"};
 
   Produces<aod::CascGood> cascgood;
-  void process(aod::Collision const& collision, aod::V0DataExt const& V0s, aod::Cascades const& Cascades,
+  void process(aod::Collision const& collision, aod::V0Datas const& V0s, aod::Cascades const& Cascades,
                soa::Join<aod::FullTracks, aod::TracksExtended> const& tracks)
   {
     for (auto& casc : Cascades) {
```

Comment:
One can also store `casc.v0()` in a variable, so that there are no repeated calls creating a new iterator.

Reply:
So in possible we should avoid subsequent calls to the content of index columns?

Reply:
Yes, especially in loops. It will always create a copy of iterator.